scripts/new_project.kts

```diff
@@ -59,38 +68,69 @@ object NewProject {
     }
 
     private fun handleArguments(args: Array<String>) {
-        val argumentError = "ERROR: Invalid argument name: Ensure define arguments => app-name={\"MyProject\"} or {\"My Project\"} package-name={com.sample.myproject}"
-        when (args.size) {
-            1 -> when {
-                args.first().startsWith(KEY_APP_NAME) -> showErrorMessage("ERROR: No package name has been provided")
-                args.first().startsWith(KEY_PACKAGE_NAME) -> showErrorMessage("ERROR: No app name has been provided")
-                else -> showErrorMessage(argumentError)
+        val argumentError = "ERROR: Invalid argument name \n$HELP_MESSAGE"
```

Comment:
We'll need to update this line too since we changed `HELP_MESSAGE` to `helpMessage` 😅